tests/test_core.py

```diff
@@ -1845,11 +1856,12 @@ def test_emscripten_api(self):
     self.set_setting('EXPORTED_FUNCTIONS', ['_main', '_save_me_aimee'])
     self.do_run_in_out_file_test('tests', 'core', 'test_emscripten_api')
 
-    # test EXPORT_ALL
-    self.set_setting('EXPORTED_FUNCTIONS', [])
-    self.set_setting('EXPORT_ALL', 1)
-    self.set_setting('LINKABLE', 1)
-    self.do_run_in_out_file_test('tests', 'core', 'test_emscripten_api')
+    if '-fsanitize=address' not in self.emcc_args:
+      # test EXPORT_ALL
+      self.set_setting('EXPORTED_FUNCTIONS', [])
+      self.set_setting('EXPORT_ALL', 1)
+      self.set_setting('LINKABLE', 1)
+      self.do_run_in_out_file_test('tests', 'core', 'test_emscripten_api')
```

Comment:
Why doesn't think work with asan?  Comment mabye?

Reply:
Adding a comment now. This is because of LINKABLE - asan doesn't support any dynamic linking or related things (emcc errors on it).